rholang/src/main/scala/rholang/rosette/Roselang.scala

```diff
@@ -139,7 +139,22 @@ extends AllVisitor[VisitorTypes.R,VisitorTypes.A] {
       }
       case StrTermPtdCtxtBr( op, subterms ) => {
         val qterms = subterms.map(( term ) => doQuote( term ))
-        B( _rx, Var( op ) :: qterms )
+        if ( op == _list )
+          B(_list, qterms)
```

Comment:
Make spacing consistent with the rest of the file.

Reply:
Done.